recipe/meta.yaml

```diff
@@ -25,20 +25,20 @@ requirements:
     - cython
     - blas 1.1 {{ variant }}
     - libgfortran  # [linux]
-    - openblas 0.2.19|0.2.19*
+    - openblas 0.2.20|0.2.20*
     - numpy 1.9.*  # [not (win and py36)]
     - numpy 1.11.*  # [win and py36]
   run:
     - python
     - blas 1.1 {{ variant }}
     - libgfortran
-    - openblas 0.2.19|0.2.19*
+    - openblas 0.2.20|0.2.20*
```

Comment:
Can you change this to `openblas >=0.2.20`?

Reply:
Only run?

Reply:
Yes. the run requirement can be relaxed that way.